src/vs/workbench/contrib/userDataSync/browser/userDataSync.ts

```diff
@@ -566,7 +564,6 @@ export class UserDataSyncWorkbenchContribution extends Disposable implements IWo
 		}, {
 			id: SyncResource.Keybindings,
 			label: getSyncAreaLabel(SyncResource.Keybindings),
-			description: this.configurationService.getValue('settingsSync.keybindingsPerPlatform') ? localize('per platform', "for each platform") : undefined
```

Comment:
Isn't this a good thing to keep?

Reply:
It would be if there is an option to toggle/configure it in the picker.